taccsite_cms/_settings/form_plugin.py

```diff
@@ -10,6 +10,14 @@
 # (No success hiding those formats from editor)
 # https://github.com/avryhof/djangocms-forms/issues/8
 
+# Unable to submit files
+# > AttributeError at /forms/submit/
+# ```
+#   File "/opt/pysetup/.venv/src/djangocms-forms-maintained/djangocms_forms/fields.py", line 42, in clean
+#     if uploaded_file._size > self.max_upload_size:
+# AttributeError: 'InMemoryUploadedFile' object has no attribute '_size'
+# ```
```

Comment:
This is not related to e-mail. I just noticed thus noted another known issue with the plugin.

Reply:
Is this the output for a file attachment attempt?

Reply:
Yes.

Seems simple enough to fix. Probably some DJANGO setting. But the image I tested was small, barely over 500KB I think. I haven't tried to fix this yet.

Reply:
Maybe we can open an issue with their library and link that here, then we could remove this commented stacktrace

Reply:
```suggestion
# Unable to submit files
# https://github.com/avryhof/djangocms-forms/issues/14
```